adala/runtimes/_litellm.py

```diff
@@ -553,9 +563,12 @@ def _get_prompt_tokens(string: str, model: str, output_fields: List[str]) -> int
         return user_tokens + system_tokens
 
     @staticmethod
-    def _get_completion_tokens(model: str, output_fields: Optional[List[str]]) -> int:
+    def _get_completion_tokens(
+        model: str, output_fields: Optional[List[str]], provider: str
+    ) -> int:
+        model, provider = resolve_litellm_model_and_provider(model, provider)
```

Comment:
why do we have to normalize this at completion time? Can we get these into a canonical format before we store them, so we don't have to deal with this here?

Reply:
store them where in LSE? Not going to work